internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider.go

```diff
@@ -200,10 +196,6 @@ func oldSDKRead(federationSettingsID, oktaIdpID string, d *schema.ResourceData,
 		return diag.FromErr(fmt.Errorf("error setting sso url (%s): %s", d.Id(), err))
 	}
 
-	if err := d.Set("id", federatedSettingsIdentityProvider.Id); err != nil {
```

Comment:
Removing this because it's equivalent of `d.SetId`

Reply:
I am not sure what do you mean with equivalent.. if it errors, aren't we removing here the return error?